package.json

```diff
@@ -1,46 +1,45 @@
 {
   "name": "@procore/js-sdk",
-  "version": "2.4.1",
+  "version": "3.0.0",
   "description": "A wrapper for the procore API",
-  "main": "lib",
+  "main": "dist/lib",
```

Comment:
When does `dist` get built? Is it created automatically when the user installs/runs the package?

This article suggests making the files in `dist` available to NPM using the `files` key:
https://mykeels.medium.com/node-how-to-handle-your-dist-files-9667f47316ab